store/Cargo.toml

```diff
@@ -30,7 +30,7 @@ miden-node-utils = { path = "../utils", version = "0.2" }
 miden-objects = { workspace = true }
 once_cell = { version = "1.18.0" }
 prost = { version = "0.12" }
-rusqlite = { version = "0.30", features = ["array", "buildtime_bindgen"] }
+rusqlite = { version = "0.30", features = ["array", "buildtime_bindgen", "bundled"] }
```

Comment:
This is the feature used by the Miden client. I am curious if the bug could be related, needing to have the same features enabled on all crates.

Reply:
which bug do you mean? This by itself wasn't a bug in itself, but it allowed one to slip through the cracks (the name mismatch on the other PR).